packages/dev/materials/src/grid/gridMaterial.ts

```diff
@@ -248,7 +249,19 @@ export class GridMaterial extends PushMaterial {
             const join = defines.toString();
             addClipPlaneUniforms(uniforms);
             subMesh.setEffect(
-                scene.getEngine().createEffect("grid", attribs, uniforms, ["opacitySampler"], join, undefined, this.onCompiled, this.onError),
+                scene.getEngine().createEffect(
+                    "grid",
+                    <IEffectCreationOptions>{

```

Comment:
Why is this cast needed?

Reply:
Good question :)

I simply copied/pasted a code that existed in `ShadowOnly`. I have corrected it.